test/component.js

```diff
@@ -63,6 +63,28 @@ describe('components', () => {
 		expect(foo).to.exist.and.have.deep.property('props.children').eql(children);
 	});
 
+	it('should single out children before componentWillReceiveProps', () => {
+		let props;
+
+		class Child extends React.Component {
+			componentWillReceiveProps(newProps) {
+				props = newProps;
+			}
+		}
+		class Parent extends React.Component {
+			render() {
+				return <Child>second</Child>;
+			}
+		}
+
+		let a = React.render(<Parent/>, scratch);
+		a.forceUpdate();
```

Comment:
I *think* `.forceUpdate()` on a parent is the easiest way to cause `componentWillReceiveProps` on a child.  If you can think of an even cleaner way to test this, let me know.